e2e/node/e2e.test.ts

```diff
@@ -111,99 +143,459 @@ const SHARED_FILE_CONTENT = "Some content.\n";
 const nodeVersion = process.versions.node.split(".");
 const nodeMajor = Number(nodeVersion[0]);
 
-const contentArr: [string, NodeFile | Buffer][] = [
-  ["buffer", Buffer.from(SHARED_FILE_CONTENT)],
-];
-
-if (nodeMajor >= 18) {
-  contentArr.push([
-    "NodeFile",
-    new NodeFile([Buffer.from(SHARED_FILE_CONTENT, "utf-8")], "text.ttl", {
-      type: "text/plain",
-    }),
-  ]);
-}
-
 async function toString(input: File | NodeFile | Buffer): Promise<string> {
   if (input instanceof Buffer) return input.toString("utf-8");
 
   return input.text();
 }
 
-describe.each(contentArr)(
-  `End-to-end access grant tests for environment [${environment}}] initialized with %s`,
-  (_, content) => {
-    const requestorSession = new Session();
-    let resourceOwnerSession: Session;
+describe(`End-to-end access grant tests for environment [${environment}]`, () => {
+  let sharedFileIri: string;
 
-    let resourceOwnerPod: string;
-    let sharedFileIri: string;
-    let vcProvider: string;
+  const requestorSession = new Session();
+  let resourceOwnerSession: Session;
+  let resourceOwnerPod: string;
+  let vcProvider: string;
 
-    // Setup the shared file
-    beforeEach(async () => {
-      // Log both sessions in.
-      await requestorSession.login({
+  async function getSharedFile() {
+    // setup the shared file
+    const savedFile = await retryAsync(() =>
+      sc.saveFileInContainer(
+        resourceOwnerPod,
+        Buffer.from(SHARED_FILE_CONTENT),
+        {
+          fetch: addUserAgent(
+            addUserAgent(resourceOwnerSession.fetch, TEST_USER_AGENT),
+            TEST_USER_AGENT,
+          ),
+          slug: v4(),
```

Comment:
Instead of putting a random slug, I would just let the server assign a resource name itself. That's functionally similar, but show a different intention if the client doesn't require a specific slug.

EDIT: I see this change comes from a dedicated commit. Is there a specific reason to have this name arbitrarily picked client-side?

Reply:
I was seeing more VCs than expected returned in https://github.com/inrupt/solid-client-access-grants-js/actions/runs/6197634526/job/16826563949?pr=741#step:6:272.

I thought it may have been because the server was re-using a name. This was not the reason. Still investigating what is happening but this commit can be reverted.